debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrPosition.java

```diff
@@ -42,7 +42,7 @@ public LcrPosition(byte[] rawPosition) {
         catch (SQLException | StreamsException e) {
             throw new RuntimeException(e);
         }
-        LOGGER.trace("LCR position {} converted to SCN {}", rawPosition, stringFromat, scn);
+        LOGGER.trace("LCR position {} converted to SCN {}", stringFromat, scn);
```

Comment:
Hi @big-andy-coates, I would prefer we take the opposite approach and introduce the extra `{}` placeholder so we can log both the hex and the raw `byte[]` for transparency.